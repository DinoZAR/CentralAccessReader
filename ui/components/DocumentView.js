import React from 'react';

import { StyleSheet, css } from '../styles/aphrodite';
import constants from '../styles/constants';

class DocumentView extends React.Component {
  render() {
    return (
      <div className={css(styles.window)}>
        <div className={css(styles.panel)}>
          <p>Some cool text to have here</p>
        </div>
      </div>
    );
  }
}

const styles = StyleSheet.create({
  window: {
    display: 'flex',
    flexDirection: 'row',
    justifyContent: 'center',
    width: '100%',
    height: '100%',
  },
  panel: {
    backgroundColor: 'black',
    color: 'white',
    maxWidth: '50rem',
    width: '100%',
    boxShadow: '0px 2px 15px -1px rgba(0,0,0,0.75)',
  }
});

export default DocumentView;
