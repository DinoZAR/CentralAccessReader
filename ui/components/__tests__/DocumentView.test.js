import React from 'react';
import DocumentView from '../DocumentView';

describe('DocumentView', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<DocumentView />);
    expect(wrapper).toMatchSnapshot();
  });
});
