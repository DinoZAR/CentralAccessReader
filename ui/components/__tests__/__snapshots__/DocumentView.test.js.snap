// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocumentView renders correctly 1`] = `
<div>
  <div>
    <p>
      Some cool text to have here
    </p>
  </div>
</div>
`;
